crates/sui-framework-tests/tests/move_tests.rs

```diff
@@ -15,36 +15,54 @@ pub(crate) const FRAMEWORK: &str = concat!(
     "/../../crates/sui-framework/packages"
 );
 
+#[cfg(not(msim))]
+const DIRS_TO_EXCLUDE: &[&str] = &[];
+/// We cannot support packages that depend on git dependencies on simtests.
+/// TODO: we probably also shouldn't be doing these in normal CI, since generally having CI depend
+/// on other git repos is frowned upon
+#[cfg(msim)]
+const DIRS_TO_EXCLUDE: &[&str] = &["nft-rental", "usdc_usage"];
```

Comment:
The issue here is that these packages depend on packages that are not in the `sui` mono-repo, so it's hard to avoid the git dependencies. The ideal fix is probably that the example code ends up in its own repo, and we sync it into the mono repo for CI purposes, and as part of that sync we fix up dependencies and do any vendoring we need to.

We should create an issue to track that work, and also involve the docs team.